.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and test
+
+# Run the jobs for pushes and pull requests targetting main branch.
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  # Job which sets up the dependencies and builds the project.
+  build:
+    name: "Build"
+    matrix:
+      os:
+        - "ubuntu-18.04"
+      python-version:
+        - "3.8"
+      go-version:
+        - "1.13.1"
```

Comment:
Can you update the Go version to be above 1.15 as per the Readme? The latest release is 1.16.2

Reply:
sounds good